daisy_workflows/image_import/windows/translate_x86.ps1

```diff
@@ -0,0 +1,308 @@
+  # Specifies that Network-Level user authentication is required.
+  New-ItemProperty -Path $registryPath -Name UserAuthentication -Value 1 -PropertyType DWORD -Force
+  # Specifies that the Transport Layer Security (TLS) protocol is used by the server and the client
+  # for authentication before a remote desktop connection is established.
+  New-ItemProperty -Path $registryPath -Name SecurityLayer -Value 2 -PropertyType DWORD -Force
+}
+
+function Enable-RemoteDesktop {
+  $ts_path = 'HKLM:\SYSTEM\CurrentControlSet\Control\Terminal Server'
+  if (-not (Test-Path $ts_path)) {
+    return
+  }
+  # Enable remote desktop in registry.
+  Set-ItemProperty -Path $ts_path -Name 'fDenyTSConnections' -Value 0 -Force
+
+  Write-Output 'Disabling Ctrl + Alt + Del.'
+  Set-ItemProperty -Path 'HKLM:\Software\Microsoft\Windows\CurrentVersion\Policies\System' -Name 'DisableCAD' -Value 1 -Force
+
+  Write-Output 'Enable RDP firewall rules.'
+  Run-Command netsh advfirewall firewall set rule group='remote desktop' new enable=Yes
+}
+
+function Install-Packages {
+  Run-Command C:\ProgramData\GooGet\googet.exe -root C:\ProgramData\GooGet -noconfirm install C:\ProgramData\GooGet\components\googet-x86.x86_32.2.16.3@1.goo
+  # Run-Command 'C:\ProgramData\GooGet\googet.exe' -root 'C:\ProgramData\GooGet' -noconfirm install googet
+  # We always install google-compute-engine-sysprep because it is required for instance activation, it gets removed later
+  # if install_packages is set to false.
+  Run-Command C:\ProgramData\GooGet\googet.exe -root C:\ProgramData\GooGet -noconfirm install C:\ProgramData\GooGet\components\google-compute-engine-powershell.noarch.1.1.1@4.goo
+  Run-Command C:\ProgramData\GooGet\googet.exe -root C:\ProgramData\GooGet -noconfirm install C:\ProgramData\GooGet\components\certgen-x86.x86_32.1.0.0@2.goo
+  Run-Command C:\ProgramData\GooGet\googet.exe -root C:\ProgramData\GooGet -noconfirm install C:\ProgramData\GooGet\components\google-compute-engine-sysprep.noarch.3.10.1@1.goo
+  # Run-Command 'C:\ProgramData\GooGet\googet.exe' -root 'C:\ProgramData\GooGet' -noconfirm install google-compute-engine-sysprep
+  if ($script:install_packages.ToLower() -eq 'true') {
+    Write-Output 'Translate: Installing GCE packages...'
+    # Install each individually in order to catch individual errors
+    Run-Command C:\ProgramData\GooGet\googet.exe -root C:\ProgramData\GooGet -noconfirm install C:\ProgramData\GooGet\components\google-compute-engine-windows-x86.x86_32.4.6.0@1.goo
+    # Run-Command 'C:\ProgramData\GooGet\googet.exe' -root 'C:\ProgramData\GooGet' -noconfirm install google-compute-engine-windows
+    # Run-Command 'C:\ProgramData\GooGet\googet.exe' -root 'C:\ProgramData\GooGet' -noconfirm install google-compute-engine-auto-updater
+    # Run-Command 'C:\ProgramData\GooGet\googet.exe' -root 'C:\ProgramData\GooGet' -noconfirm install google-compute-engine-vss -ErrorAction SilentlyContinue
```

Comment:
Eventually we will install and manage packages using a googet repository. Since we do not currently have a repository to distribute packages from, I need to do the equivalent of a local package install.

Reply:
In general, the best practice is not to publish code with commented out code. Use your best judgement if you feel we should not follow that rule here.

Reply:
I'm going to take @EricEdens advice and recombine this file, which will have the added benefit of removing the need to comment these files out, since I will need to write a separate function for 32-bit package installs.

Reply:
I have combined the PowerShell files, which resulted in adding logic to detect a 64-bit or 32-bit import workflow, and choose to install packages from a repository (64-bit) or locally (32-bit).